requirements.txt

```diff
@@ -1,14 +1,13 @@
-absl-py
-packaging
-pandas
-tensorflow
-tensorflow-datasets
-flake8
-regex
-isort
-black
-pytest
-pycocotools
-keras-core
-build
-namex
+# Tensorflow.
+tf-nightly-cpu==2.16.0.dev20231109  # Pin a working nightly until rc0.
+keras-nightly==3.0.0.dev2023111903  # TODO recent nightly fails keras_cv/models/object_detection/retinanet/retinanet_test.py
```

Comment:
typo with the path pasted here?

Reply:
Thats how it's in PyPi - https://pypi.org/project/keras-nightly/3.0.0.dev2023111903/
The reason for it to look off is we may be saving down to Minute and TF only getting name down to the hour.

Reply:
oh sorry my bad, the line was wrapping on `keras_cv/models/object_detection/retinanet/retinanet_test.py` so i didn't realize it was part of the comment for the failure